### ISSUE TITLE
Cannot build.

### ISSUE THREAD

**Issue by prange** [material-components/material-components-web-components#170]
Following instructions to build master (as of today)

`git clone`
`npm run bootstrap`

fails with (npm log)
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/Cellar/node/11.3.0_1/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'run',
1 verbose cli   'build-typescript' ]
2 info using npm@6.4.1
3 info using node@v11.3.0
4 verbose run-script [ 'prebuild-typescript',
4 verbose run-script   'build-typescript',
4 verbose run-script   'postbuild-typescript' ]
5 info lifecycle material-components-web-components@~prebuild-typescript: material-components-web-components@
6 info lifecycle material-components-web-components@~build-typescript: material-components-web-components@
7 verbose lifecycle material-components-web-components@~build-typescript: unsafe-perm in lifecycle true
8 verbose lifecycle material-components-web-components@~build-typescript: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:~/workspace/material-components-web-components/node_modules/.bin:~/Downloads/google-cloud-sdk/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:~/bin
9 verbose lifecycle material-components-web-components@~build-typescript: CWD: ~/workspace/material-components-web-components
10 silly lifecycle material-components-web-components@~build-typescript: Args: [ '-c', 'tsc -b --verbose packages/**/tsconfig.json' ]
11 silly lifecycle material-components-web-components@~build-typescript: Returned: code: 1  signal: null
12 info lifecycle material-components-web-components@~build-typescript: Failed to exec build-typescript script
13 verbose stack Error: material-components-web-components@ build-typescript: `tsc -b --verbose packages/**/tsconfig.json`
13 verbose stack Exit status 1
```

Excerpt from output:
```
[5:41:34 PM] Project 'packages/base/tsconfig.json' is out of date because output file 'packages/base/base-element.js' does not exist

[5:41:34 PM] Building project '/Users/atlosm/workspace/material-components-web-components/packages/base/tsconfig.json'...

packages/base/src/base-element.ts:22:24 - error TS2307: Cannot find module 'lit-html/directives/classMap.js'.

22 export {classMap} from 'lit-html/directives/classMap.js';
                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

[5:41:35 PM] Project 'packages/button/tsconfig.json' is out of date because output file 'packages/button/mwc-button-css.js' does not exist

[5:41:35 PM] Building project '/Users/atlosm/workspace/material-components-web-components/packages/button/tsconfig.json'...

packages/button/src/mwc-button.ts:18:24 - error TS2307: Cannot find module 'lit-html/directives/classMap.js'.

18 import {classMap} from 'lit-html/directives/classMap.js';
                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

I get instructed to install event-stream and babel manually by npm first (which i do)

Any hints on how to advance on this?

thank you.

**Comment by jtbrne** [comment]
Latest commit fixed this.

**Comment by straversi** [comment]
Fixed by fbae6bedd5d8fcf5657e0acb2764a970eec2003c
